### PR TITLE
Ratings - Bug Fixes - change request stubs- css 

### PR DESCRIPTION
**Pull request by ryanaolea**
This PR covers all the bug fixes found in new review form. There is now styling for form buttons and cloudinary is set up for a user to submit photos  in form. 
I also stubbed out the functions we will need for the change request. 
If you see any console.logs please make note of them and i will comment them out but am using still for testing purposes.

Things left to do for Rating:
1. Add event listener to stars vote that renders description of star on click - this is for the form modal stars
2. refactor upload image to render thumbnail instead of button on submit to cloudinary

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -35,25 +51,35 @@ class App extends React.Component {
...
+
   render () {
+    window.addEventListener('click', function (e) {
+      let time = new Date().toLocaleString();
+      let body = {
+        time: time,
+        module: findId(e.target.parentNode),
+        element: e.target.outerHTML
+      };
+      console.log('this is your click event', body);
```
Comment (Daniel-Ghaly) [inline client/src/App.jsx:64]: leaving this for testing?

File: client/src/components/RatingsComponents/ReviewsList/HelpfulTile.jsx [hunk 2]
```diff
@@ -32,11 +35,33 @@ function HelpfulTile ({helpfulnessCount, reviewId, setReviewsCount}) {
...
+  //     method: 'put',
+  //     url: `http://127.0.0.1:3000/reviews/${reviewId}/report`,
+  //     params: {
+  //       review_id: reviewId
+  //     }
+  //   })
+  //     .catch((err) => console.error('error putting report::::', err));
+  //   setReviewsCount(prevState => prevState - 1);
+  // };
+
```
Comment (Daniel-Ghaly) [inline client/src/components/RatingsComponents/ReviewsList/HelpfulTile.jsx:64]: should we remove this, or will you still need it?

File: client/src/components/RatingsComponents/ReviewsList/NewReviewForm.jsx [hunk 3]
```diff
@@ -295,4 +346,12 @@ function NewReviewForm ({factors, productName, closeModalOnSubmit, characteristi
...
\ No newline at end of file
+//for form submission will need to use : e.target.getAttribute("data-stars"); to send star vote to api
+
+{ /* <button className="form__photo"
+            onClick={() => addPhotosInputBox()}
+            id="submit-photo"
+            type="button"
+            value="submit-photo">
+            Upload Your Photos
+          </button> */ }
```
Comment (Daniel-Ghaly) [inline client/src/components/RatingsComponents/ReviewsList/NewReviewForm.jsx]: still need this?

File: server.js [hunk 4]
```diff
@@ -30,7 +35,8 @@ const axiosPut = (path, body) => {
 };
 
 const axiosPost = (path, body) => {
-  axios.post(`${url}${path}`, body, { headers })
+  console.log('this is the axiospost:::', body);
```
Comment (Daniel-Ghaly) [inline server.js]: can remove

**Review by Daniel-Ghaly — APPROVED** [review]
just some minor stuff but I approved it so you can merge it right away after you makes those changes

File: server.js (hunk 4 above)
Comment (ryanaolea) [reply server.js]: ✅

File: client/src/App.jsx (hunk 1 above)
Comment (ryanaolea) [reply client/src/App.jsx:64]: yes, until we have the axios post that goes right after this and i wanted yall to see what the click events actually register so we can use that ifo in the proper params for post to api.

File: client/src/components/RatingsComponents/ReviewsList/HelpfulTile.jsx (hunk 2 above)
Comment (ryanaolea) [reply client/src/components/RatingsComponents/ReviewsList/HelpfulTile.jsx:64]: leaving for refactoring, idea is that all the requests be routed through server.js eventually

**Comment by ryanaolea** [comment]
Thank you for all the feedback and for all the masterful bug hunting. Go team Jessie!!!!!

File: client/src/components/RatingsComponents/ReviewsList/NewReviewForm.jsx (hunk 3 above)
Comment (ryanaolea) [reply client/src/components/RatingsComponents/ReviewsList/NewReviewForm.jsx]: nope! nice catch!

**Pull request merged**